debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/spi/Snapshotter.java

```diff
@@ -63,6 +63,8 @@ default boolean shouldStreamEventsStartingFromSnapshot() {
      */
     Optional<String> buildSnapshotQuery(TableId tableId);
```

Comment:
Since the `Snapshotter` interface is still `@Incubating`, I would recommend removing this old method in favor of the new.